health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/OldCampaignCard.js

```
@@ -3,39 +3,38 @@
-  BOUNDARY_MANAGER:["BOUNDARY_MANAGER"],
-  CAMPAIGN_MANAGER_ONLY:["CAMPAIGN_MANAGER"],
-  NATIONAL_SUPERVISOR:["NATIONAL_SUPERVISOR"]
+  CAMPAIGN_MANAGER: ["CAMPAIGN_MANAGER", "MICROPLAN_CAMPAIGN_INTEGRATOR"],
+  BOUNDARY_MANAGER: ["BOUNDARY_MANAGER"],
+  CAMPAIGN_MANAGER_ONLY: ["CAMPAIGN_MANAGER"],
+  NATIONAL_SUPERVISOR: ["NATIONAL_SUPERVISOR"],
 };
 
 /* @nipun delete this card once migrated  */
 const CampaignCard = () => {
   if (!Digit.Utils.didEmployeeHasAtleastOneRole(Object.values(ROLES).flatMap((e) => e))) {
```

Biome (1.9.4)

[error] Avoid unnecessary callback in flatMap call.

You can just use flat to flatten the array.
Safe fix: Replace unnecessary flatMap call to flat instead.


(lint/correctness/noFlatMapIdentity)